README.md

```diff
@@ -0,0 +1,168 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>My first microverse project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [your_project_name] <a name="about-project">my first microverse project</a>
+
+hello microverse is my first microverse project as a microverse student
+
+## 🛠 Built With <a name="built-with">html and css</a>
+
+### Tech Stack <a name="tech-stack">html and css</a>
+
+i built this web display with html and css styling 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.org/">html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>css</summary>
+  <ul>
+    <li><a href="https://css.com/">css</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features">key features</a>
+
+> Describe between 1-3 key features of the application.
+
```

Comment:
Very well done so far , one small thing is that you need to remove default text under each section after adding your own content